docker-compose-dev.yml

```diff
@@ -50,7 +51,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.dev
-    command: coordinator -f test_array.npy
+    command: coordinator -f test_array.npy --storage-endpoint http://minio-dev:9000 --storage-key-id $${MINIO_ACCESS_KEY} --storage-secret-access-key $${MINIO_SECRET_KEY} --storage-bucket xain-fl-aggregated-weights
```

Comment:
During the build of the docker images I get the following error:
```
Running command git clone -q https://github.com/xainag/xain-proto.git /tmp/pip-install-jvk79hx_/xain-proto
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?
```

Reply:
It should be fixed now, could you give it another try?

Reply:
I also made `pip install` verbose, otherwise it may seem like the command is hanging.

Reply:
cc @dkravetz  for the docker files changes.

Reply:
There is something not right with `$${MINIO_SECRET_KEY}` and `$${MINIO_ACCESS_KEY}`. 
I get: 

```
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "Exception calling application: An error occurred (InvalidAccessKeyId) when calling the PutObject operation: The access key ID you provided does not exist in our records."
	debug_error_string = "{"created":"@1579007385.100986000","description":"Error received from peer ipv4:127.0.0.1:50051","file":"src/core/lib/surface/call.cc","file_line":1056,"grpc_message":"Exception calling application: An error occurred (InvalidAccessKeyId) when calling the PutObject operation: The access key ID you provided does not exist in our records.","grpc_status":2}"
```

But it works when I use the hardcoded values `minio` and `minio123`

Reply:
oh I actually simply forgot to add the `environment` section that set those variables...